src/components/extension-landing/extension-landing.scss

```diff
@@ -39,7 +39,7 @@
     .headline-icon {
         height: 40px;
         width: 40px;
-        margin: auto;
+        margin: auto 0;
```

Comment:
this is the key change: we don't want to use horizontal margin, just vertical margin.